### PR TITLE
Earth 12 hero

### PR DESCRIPTION
**Pull request by koppieesq**
# READY FOR REVIEW

# Summary
Adds hero banner for Stanford Earth

# Needed By (Date)
Monday 5/22

# Steps to Test

1. Create "hero banner" paragraph
2. Add fields & configure display
3. Insert beautiful new hero banner into content

# Affected versions or modules
Corresponding theme PR: https://github.com/stanford-earth/matson/pull/4

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/EARTH-12
- @sherakama

### CODE REVIEW

**Comment by koppieesq** [comment]
I noticed a couple odd changes also got added:

1. Minor change to package.json, don't know why (npm updated something?)
2. Added shadow definition to rendered css.  I'm guessing some sass had previously been saved but not compiled?

**Comment by koppieesq** [comment]
@sherakama I missed your note about the gruntfile, sorry about that!  I added the entires.

I was using Grunt to compile, as requested.  It was still able to compile the sass, even without the entires in the gruntfile, which seems odd in retrospect.  /shrug

Anyway, I believe I've completed all your requested changes now.  This is ready for your review.

File: Gruntfile.js [hunk 1]
```diff
@@ -79,20 +79,22 @@ module.exports = function(grunt) {
...
           'patterns/molecules/postcard/css/postcard.theme.css':                               'patterns/molecules/postcard/scss/postcard.theme.scss',
           'patterns/molecules/hero-with-overlay/css/hero-with-overlay.component.css':         'patterns/molecules/hero-with-overlay/scss/hero-with-overlay.component.scss',
           'patterns/molecules/hero-with-overlay/css/hero-with-overlay.theme.css':             'patterns/molecules/hero-with-overlay/scss/hero-with-overlay.theme.scss',
           'patterns/molecules/secondary-sidebar-nav/css/secondary-sidebar-nav.component.css': 'patterns/molecules/secondary-sidebar-nav/scss/secondary-sidebar-nav.component.scss',
-          'patterns/molecules/secondary-sidebar-nav/css/secondary-sidebar-nav.theme.css':     'patterns/molecules/secondary-sidebar-nav/scss/secondary-sidebar-nav.theme.scss'
+          'patterns/molecules/secondary-sidebar-nav/css/secondary-sidebar-nav.theme.css':     'patterns/molecules/secondary-sidebar-nav/scss/secondary-sidebar-nav.theme.scss',
+          'patterns/molecules/hero-banner/css/hero-banner.component.css':                     'patterns/molecules/hero-banner/css/hero-banner.component.css',
```
Comment (sherakama) [inline Gruntfile.js]: I cannot get this one to compile using the built in grunt tool.

**Comment by sherakama** [comment]
@koppieesq 

As per our conversation last Friday please do the following

1. Clean up the SCSS to ensure it compiles by creating a new utilities directory that matches the matson theme's in stanford_components or by adding to Decanter.
2. Fix the stroke dropping out of the image area
3. Clean up duplication between theme (matson) scss and stanford_component scss. Right now they are copies of each other and the component should have structural SCSS and the theme (matson) should have what makes it look like Earth.

**Pull request merged**